fam_change_permissions.html

```diff
@@ -161,7 +169,7 @@ <h2>Example</h2>
 		</article>
 	</section>
 	<footer>
-		<p>Copyright 2019, Hewlett Packard Enterprise Development Co, LLP</p>
+		<p>Copyright 2021, Hewlett Packard Enterprise Development Co, LLP</p>
```

Comment:
Delete three lines starting at 159 (// free allocated space in FAM) in this example. If the fam is deallocated, the change in permissions is meaningless for the next program.